tests/beacon/test_aggregation.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from hypothesis import (
+    given,
+    settings,
+    strategies as st,
+)
+
+from eth.utils import bls
+from eth.utils.bitfield import (
+    get_empty_bitfield,
+    has_voted,
+)
+from eth.beacon.aggregation import (
+    aggregate_votes,
+    verify_votes,
+)
+
+
+@settings(max_examples=1)
+@given(random=st.randoms())
+@pytest.mark.parametrize(
+    (
+        'votes_count'
+    ),
+    [
+        (0),
+        (9),
+    ],
+)
+def test_aggregate_votes(votes_count, random, privkeys, pubkeys):
+    bit_count = 10
+    pre_bitfield = get_empty_bitfield(bit_count)
+    pre_sigs = ()
+
+    random_votes = random.sample([i for i in range(bit_count)], votes_count)
```

Comment:
nitpick: could `[i for i in range(bit_count)]` be replaced with ` range(bit_count)`?
```python
>>> import random
>>> random.seed(5566)
>>> random.sample([i for i in range(10)], 5)
[0, 9, 2, 8, 4]
>>> random.seed(5566)
>>> random.sample(range(10), 5)
[0, 9, 2, 8, 4]
```